device_doctor/bin/main.dart

```diff
@@ -11,7 +11,7 @@ import 'package:device_doctor/device_doctor.dart';
 const String actionFlag = 'action';
 const String deviceOSFlag = 'device-os';
 const String helpFlag = 'help';
-const List<String> supportedOptions = <String>['healthcheck', 'recovery', 'propertycheck'];
+const List<String> supportedOptions = <String>['healthcheck', 'recovery', 'properties'];
```

Comment:
nit: Moving these to static variables would allow for reuse, eg `kHealthCheckCommand`

Reply:
IIRC, `static` is used to implement class-wide variables and methods. Will extend this to more general like a const file when needed.